config/config.go

```diff
@@ -258,6 +260,14 @@ type AgentConfig struct {
 	// the state in the file will be restored for the plugins.
 	Statefile string `toml:"statefile"`
 
+	// PersistInterval The interval in which the state will be stored into the
+	// statefile. When set to 0, persist interval is disabled.
+	PersistInterval Duration `toml:"statefile_persist_interval"`
+
+	// RoundPersistInterval rounds persist interval to 'interval'.
+	//     ie, if Interval=10s then always persist state on :00, :10, :20, etc.
```

Comment:
```suggestion
	// RoundPersistInterval rounds persist interval to 'PersistInterval'.
	//     ie, if PersistInterval=10m then always persist state on :00, :10, :20, etc.
```

Reply:
Should be resolved as of this [commit](https://github.com/influxdata/telegraf/pull/14561/commits/35de14b84db306904d5376dbea7f3f27ece47a95)